code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -135,22 +142,31 @@
 		. = ..()
 
 /obj/effect/alien/resin/special/pylon/core/Destroy()
+
 	if(linked_hive)
 		linked_hive.hive_location = null
+		if(world.time < HIVECORE_COOLDOWN_CUTOFF && !hardcore)
+			. = ..()
+			return
+		linked_hive.hivecore_cooldown = TRUE
+		xeno_message(SPAN_XENOANNOUNCE("A sudden tremor ripples through the hive... the Hive Core has been destroyed! Vengeance!"), 3, linked_hive.hivenumber)
```

Comment:
redundant message as it already gives you this : ` Hive: A hive core has been destroyed at Open grounds (event P)!`